Source/ACE.Server/WorldObjects/WorldObject_Properties.cs

```diff
@@ -3059,6 +3059,17 @@ public bool DontTurnOrMoveWhenGiving
             set { if (!value) RemoveProperty(PropertyBool.DontTurnOrMoveWhenGiving); else SetProperty(PropertyBool.DontTurnOrMoveWhenGiving, value); }
         }
 
+        /// <summary>
+        /// Determines the rotation speed for projectiles in global X
+        /// </summary>
+        public double? RotationSpeed
+        {
+            get => GetProperty(PropertyFloat.RotationSpeed);
+            set { if (!value.HasValue) RemoveProperty(PropertyFloat.RotationSpeed); else SetProperty(PropertyFloat.RotationSpeed, value.Value); }
+        }
+
+        public bool HasMissileFlightPlacement => CSetup.PlacementFrames.ContainsKey((int)ACE.Entity.Enum.Placement.MissileFlight);
```

Comment:
seems like this would be worth caching somewhere into a simple bool. doing a dictionary lookup for every projectile launched is not exactly ideal